apps/web/playwright/filters.e2e.ts

```diff
@@ -0,0 +1,696 @@
+    await page.keyboard.press("Enter");
+
+    await expect(page.getByText("Response 1")).toBeVisible();
+    await expect(page.getByText("Response 2")).toBeHidden();
+
+    await prisma.booking.delete({ where: { id: booking1.id } });
+    await prisma.booking.delete({ where: { id: booking2.id } });
+  });
+
+  test("TEXT field: should filter with different text operators", async ({
+    page,
+    users,
+    routingForms,
+    prisma,
+  }) => {
+    const owner = await users.create(undefined, {
+      hasTeam: true,
+      isUnpublished: true,
+      isOrg: true,
+      hasSubteam: true,
+    });
+    await owner.apiLogin();
+    const membership = await owner.getOrgMembership();
+
+    const textFieldId = uuidv4();
+    const formName = "Text Filter Test Form";
+    const form = await routingForms.create({
+      name: formName,
+      userId: owner.id,
+      teamId: membership.teamId,
+      fields: [
+        {
+          type: "text",
+          label: "Description",
+          identifier: "description",
+          required: true,
+        },
+      ],
+    });
+
+    await prisma.app_RoutingForms_FormResponse.create({
+      data: {
+        formFillerId: "test-filler-1",
+        formId: form.id,
+        response: {
+          [textFieldId]: {
+            label: "Description",
+            value: "This is a test description",
+          },
+        },
+      },
+    });
+
+    await prisma.app_RoutingForms_FormResponse.create({
+      data: {
+        formFillerId: "test-filler-2",
+        formId: form.id,
+        response: {
+          [textFieldId]: {
+            label: "Description",
+            value: "Another description for testing",
+          },
+        },
+      },
+    });
+
+    await prisma.app_RoutingForms_FormResponse.create({
+      data: {
+        formFillerId: "test-filler-3",
+        formId: form.id,
+        response: {
+          [textFieldId]: {
+            label: "Description",
+            value: "",
+          },
+        },
+      },
+    });
+
+    await page.goto(`/insights/routing`);
+
+    await selectFilter(page, "description");
+    await page.getByTestId("filter-popover-trigger-description").click();
+    await page
+      .getByTestId("select-filter-options-description")
+      .getByRole("option", { name: "contains" })
+      .click();
+    await page.getByPlaceholder("Filter value").fill("test");
+    await page.keyboard.press("Enter");
+
+    await expect(page.getByText("This is a test description")).toBeVisible();
+    await expect(page.getByText("Another description for testing")).toBeVisible();
+
+    await clearFilters(page);
+    await selectFilter(page, "description");
+    await page.getByTestId("filter-popover-trigger-description").click();
+    await page
+      .getByTestId("select-filter-options-description")
+      .getByRole("option", { name: "equals" })
+      .click();
+    await page.getByPlaceholder("Filter value").fill("This is a test description");
```

Comment:
Change case to see if the filter works regardless of case (the filter is supposed to be case insensitive)